src/test/java/trello/ui/main.java

```diff
@@ -5,41 +5,84 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.junit.Assert;
 
 import java.util.List;
 
 public class main {
+
+    private static void waitForPageLoaded(WebDriver driver, long sleep) {
```

Comment:
@ArturoAmoroso it's not really necessary to wait for the loading of page since the webdriver does it internally and even when the document goes to complete could be some elements which are being created after this.
The better for these cases is using waits from selenium.